src/services/neuron/neuronApi.ts

```diff
@@ -48,6 +50,32 @@ export const sendCyberlink = async (
   return transactionHash;
 };
 
+export const sendCyberlinkArray = async (
```

Comment:
this definitely not `neuron` entity,  shouldn't be here.
soft3 don't have this method?